packages/cli/src/commands/versions/bump.ts

```diff
@@ -372,51 +372,64 @@ export function createVersionFinder(options: {
     releaseManifest?.packages.map(p => [p.name, p.version]),
   );
   return async function findTargetVersion(name: string) {
-    const existing = found.get(name);
+    let _name = name;
```

Comment:
Avoid `_` prefix except for unused parameters, this could be given a more descriptive name instead